File: shop-online/service-catalogue/src/main/java/service/catalogue/entities/ProductGroup.java
package service.catalogue.entities;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Table;

import core.dao.entities.BaseEntity;

@Entity
@Table(name="product_groups")
public class ProductGroup extends BaseEntity {
	private static final long serialVersionUID = -3065264917043198173L;

	@Column(name = "name", columnDefinition = MEDIUM_1)
	private String name;
	
	@Column(name = "description", columnDefinition = MEDIUM_5)
	private String description;
	
	public String getName() {
		return name;
	}
	
	public void setName(String name) {
		this.name = name;
	}
	
	public String getDescription() {
		return description;
	}
	
	public void setDescription(String description) {
		this.description = description;
	}
}

File: shop-online/web-spa-business/src/app/product/shared/models/product.group.ts
export class ProductGroup {
  id: string;
  name: string;
  note: string;
  status: string;

  constructor(id?: string, name?: string, note?: string, status?: string) {
    if (id) {
      this.id = id;
    } else {
      this.id = '-1';
    }
    this.name = name;
    this.status = status;
    this.note = note;

  }
}